imagej.html

```diff
@@ -247,85 +189,43 @@ <h2><a class="anchor" id="imagej5"></a>Using ImageJ with OMERO</h2>
 </li>
 
 <li>
-<p class="first">Save or move the .zip archive to the <span class="code-example">ImageJ &gt; plugins</span> folder.</p>
-<p>If upgrading from a previous version, delete all old versions of the plugin components from the folder before going any further.</p>
-<p>Extract the archive into the plugins folder to give a folder named &#8220;OMERO.insight-ij-x&#8221; where x is the version number.</p>
-<p>Leave the complete <span class="code-example">OMERO.insight-ij</span> plugin folder with its components in the <span class="code-example">ImageJ &gt; plugins</span> folder.</p>
+<p class="first">Extract the downloaded .zip archive. Remember to where did you extract it to. Copy the extracted folder and paste it to the Plugins folder of your Fiji.
 
 <div class="note">
 <p class="note-title">Note</p>
-<p class="note-text">Some Windows unzip apps create a double folder enclosing the plugin.<br />
-Check, and if this has happened, copy the inner <span class="code-example">OMERO.insight-ij-5.x.x</span> folder into the <span class="code-example">ImageJ &gt; plugins</span> folder.</p>
+<p class="note-text">Some Windows unzip apps create a double folder enclosing the plugin.<br />Check, and if this has happened, copy the inner <span class="code-example">OMERO.insight-ij-5.x.x</span> folder into the <span class="code-example">Fiji.app &gt; plugins</span> folder.</p>
 </div><!-- End note -->
 
-<div class="line-block">
-<div class="line"><br /></div>
-</div>
-<img alt="../images/imagejOmeroPlugins4.jpg" class="align-center" src="images/imagejOmeroPlugins5.jpg" style="width: 100%;" />
-<div class="line-block">
-<div class="line"><br /></div>
-</div>
-
-<p>The <span class="code-example">OMERO.insight-ij-5.x.x.zip</span> archive can be left in the plugins folder or deleted.</p>
+<p>The location of the Plugins folder vary according to where did you install your Fiji or ImageJ. The paths in the screenshots below are just examples.:</p>
```

Comment:
"... to where did you install ..." should become "... to where you installed ..."